### DISCUSSION TITLE
Height Overlay

### DISCUSSION THREAD

**Discussion by GamerInspired** [voxeltycoon/issues#483] [General]
When building I find it hard to see the difference in terrain levels.

To solve this you could have a height overlay either with numbers or alternating colours.

Or have alternating colours in build mode by default.

See circled area; it's a hill be its hard to tell at a glance where the levels change on the reverse side.
![terrainlevels](https://user-images.githubusercontent.com/8201304/60357202-ebcc7780-99ca-11e9-8c2b-9628a019accf.png)

**Comment by katepolak** [comment]
This would be quite useful indeed.

A technical note for the devs:
This would be quite easy to do in the white mode shader by alternating two colours based on the world position uniform provided by Unity. Wouldn't require any changes to the geometry.
